rhizome/lib/vm_setup.rb

```diff
@@ -542,7 +587,7 @@ def install_systemd_unit(max_vcpus, cpu_topology, mem_gib, vhost_sockets)
 --console off --serial file=#{vp.serial_log} \
 --cpus #{cpu_setting} \
 --memory size=#{mem_gib}G,hugepages=on,hugepage_size=1G \
---net "mac=#{guest_mac},tap=tap#{@vm_name},ip=,mask="
+#{net_params.join("\n")}
```

Comment:
seems like you want a line continuation mark (`\`) in the join, not just newline.

Reply:
Thanks for catching this. I corrected it.